### ISSUE TITLE
Add dark/light mode option ?

### ISSUE THREAD

**Issue by thefrcrazy** [manueljlz/gcphone#91]
Hello possible to add an option dark or light mode, because from my rating I pass them all in dark but the best would be the option
![Screenshot_40](https://user-images.githubusercontent.com/37953895/87786685-acbdce00-c83a-11ea-888a-9fa5980b8856.png)

**Comment by manueljlz** [comment]
Yes, but later I will give a little more information than is at hand.🤫🤫
